src/commonMain/kotlin/io/ktor/io/ByteArrayBufferPool.kt

```diff
@@ -4,7 +4,7 @@ public class ByteArrayBufferPool(
     capacity: Int = DEFAULT_POOL_CAPACITY,
 ) : DefaultPool<ByteArrayBuffer>(capacity) {
```

Comment:
may be also will be useful to allow creating buffer pool from `ByteArrayPool`?

Reply:
There is a constructor for that

Reply:
Not sure I understand what you mean here, I don't see such constructor

Reply:
Fixed